Jenkinsfile.integration

```diff
@@ -11,6 +11,11 @@ pipeline {
             label "cloud-ccp-ci"
         }
     }
+
+    parameters {
+        choice(choices: ['osh', 'airship'], defaultValue: 'osh', description: 'Which deployment mechanism?', name: 'deployment')
```

Comment:
Didn't we want `airship` as default deployment path?

Reply:
Not sure how the discussion ended. I agree to set 'airship' as default. But afaik, `airship' is broken...

Reply:
Done